expression/builtin_time.go

```diff
@@ -4687,7 +4687,7 @@ type builtinUTCTimeWithoutArgSig struct {
 // See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_utc-time
 func (b *builtinUTCTimeWithoutArgSig) evalDuration(row []types.Datum) (types.Duration, bool, error) {
 	// the types.ParseDuration here would never fail, so the err returned can be ignored.
-	v, _ := types.ParseDuration(time.Now().UTC().Format(types.TimeFormat), 0)
+	v, _, _ := types.ParseDuration(time.Now().UTC().Format(types.TimeFormat), 0)
```

Comment:
why ignore `isNull` and `err` returned by `types.ParseDuration` ?

Reply:
@zz-jason Input parameters can be predicted, so ignore is safe, just like the origin comments said.
But the modifications are minor, and I can modify it.